azure-spring-boot-parent/pom.xml

```diff
@@ -242,27 +260,51 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>cobertura-maven-plugin</artifactId>
-                <version>2.7</version>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
```

Comment:
Cobertura has not been maintaines since 2015 and is Java 11 incompatible.
Replaced it by JaCoCo, which is still maintained and officially supports Java11+.